wapitiCore/attack/mod_cookieflags.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+
+class mod_cookieflags(Attack):
+
+    name = "cookieflags"
+
+    def check_secure_flag(self, cookie: object):
+
+        return cookie.secure
+
+    def check_httponly_flag(self, cookie: object):
+
+        return 'HttpOnly' in cookie._rest
+
+    def attack(self):
+        url = self.persister.get_root_url()
+        session = requests.session()
+        _req = session.get(url)
```

Comment:
You should at least use the proxy settings and auth params set by the user on the command line.

Reusing the crawler instance (`self.crawler`) and iterating over `crawler.session_cookies` you should be able to have the safe results without even doing a HTTP request while using all required proxy/auth settings.

Reply:
Fixed